### PR TITLE
bt-527: Create endpoint to get short user info

### LINKED ISSUE #527: Create endpoint to get talents/employers in admin dashboard
Create endpoint to get short data for this list
Please implement GET endpoint with querystring param userType for employers and talents to not to duplicate code

- [x] create query to get only necessary data
- [x] create DTO
- [x] create repository/service methods
- [x] create endpoint

Image

### PR DESCRIPTION
**Pull request by ptarho**
image
image

### CODE REVIEW

File: backend/src/bundles/user-details/user-details.repository.ts [hunk 1]
```diff
@@ -64,6 +64,18 @@ class UserDetailsRepository implements Repository {
         });
     }
 
+    public async findUnconfirmedByRole(
+        role: 'talent' | 'employer',
+    ): Promise<UserDetailsModel[]> {
+        return await this.userDetailsModel
```
Comment (MaksymGavatuk) [inline backend/src/bundles/user-details/user-details.repository.ts]: Don't use return await

```suggestion
        return this.userDetailsModel
```

File: backend/src/bundles/user-details/user-details.repository.ts [hunk 2]
```diff
@@ -64,6 +64,18 @@ class UserDetailsRepository implements Repository {
...
+            .query()
+            .joinRelated('user')
+            .leftOuterJoinRelated('photo')
+            .where('user.role', role)
+            .andWhere('isApproved', false)
+            .select('user_id', 'photo.url as photoUrl', 'full_name');
```
Comment (MaksymGavatuk) [inline backend/src/bundles/user-details/user-details.repository.ts]: add execute method

**Review by MaksymGavatuk — CHANGES_REQUESTED** [review]

File: backend/src/bundles/user-details/user-details.repository.ts (hunk 1 above)
Comment (ptarho) [reply backend/src/bundles/user-details/user-details.repository.ts]: fixed

File: backend/src/bundles/user-details/user-details.repository.ts (hunk 2 above)
Comment (ptarho) [reply backend/src/bundles/user-details/user-details.repository.ts]: fixed

**Comment by nikita-remeslov** [comment]
coflicts

**Review by nikita-remeslov — APPROVED** [review]

**Review by MaksymGavatuk — APPROVED** [review]

**Pull request merged**